docs/openapi/docs.go

```diff
@@ -0,0 +1,163 @@
+                ],
+                "tags": [
+                    "Schema Review"
+                ],
+                "summary": "Check the SQL statement.",
+                "parameters": [
+                    {
+                        "type": "string",
+                        "description": "The environment name",
+                        "name": "env",
+                        "in": "query",
+                        "required": true
+                    },
+                    {
+                        "type": "string",
+                        "description": "The SQL statement",
+                        "name": "statement",
+                        "in": "query",
+                        "required": true
+                    },
+                    {
+                        "enum": [
+                            "MySQL",
+                            "PostgreSQL",
+                            "TiDB"
+                        ],
+                        "type": "string",
+                        "description": "The database type",
+                        "name": "database",
+                        "in": "query",
+                        "required": true
+                    },
+                    {
+                        "type": "string",
+                        "default": "utf8mb4",
+                        "description": "The database charset. Default 'utf8mb4'",
+                        "name": "charset",
+                        "in": "query"
+                    },
+                    {
+                        "type": "string",
+                        "default": "utf8mb4_general_ci",
+                        "description": "The database collation. Default 'utf8mb4_general_ci'",
+                        "name": "collation",
+                        "in": "query"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/api.SQLCheckResult"
+                        }
+                    },
+                    "400": {
+                        "description": "Bad Request",
+                        "schema": {
+                            "$ref": "#/definitions/echo.HTTPError"
+                        }
+                    },
+                    "500": {
+                        "description": "Internal Server Error",
+                        "schema": {
+                            "$ref": "#/definitions/echo.HTTPError"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "advisor.Advice": {
+            "type": "object",
+            "properties": {
+                "code": {
+                    "type": "integer"
+                },
+                "content": {
+                    "type": "string"
+                },
+                "status": {
```

Comment:
Are we able to declare ENUM?

Reply:
I'm also trying to find ways to let swagger generate the enum type for status, but haven't succeeded yet. I also try to add enum declare manually but seems cannot work.🤔 For now just update the comment for it.